### ISSUE TITLE
Allow user to set the default number of tracks he wants at start; and set the default value at 2

### ISSUE THREAD

**Issue by raindropsfromsky** [OpenShot/openshot-qt#5434]
OpenShot starts with five tracks.
For most beginners, this is too complex. They would need two or three tracks at most.
So, the app should start with 2 tracks as default, and let the user more tracks as he needs.
This will reduce the clutter.

Also, let the number of tracks be a user-selectable option in the **Preferences** dialog.
Again, its default value should be 2.

**Comment by Colorjet3** [comment]
Agreed.  I believe this will be addressed during the UI redesign face.  This is one of those low priority things.
